docs/azure_arc_jumpstart/azure_arc_sqlsrv/azure/azure_arm_template_sqlsrv_defender/_index.md

```diff
@@ -0,0 +1,269 @@
+
+  ![Screenshot showing Azure Arc-enabled SQL resources](./post-deployment-portal-rg.png)
+
+  ![Screenshot showing Azure Arc-enabled server](./post-deployment-portal-arc.png)
+
+  ![Screenshot showing Azure Arc-enabled SQL server](./post-deployment-portal-arcsql.png)
+
+  ![Screenshot showing Azure Arc-enabled SQL server threat simulation](./post-deploy-script-testsqlinjection.png)
+
+- Open Hyper-V to login to nested SQL server VM
+
+  ![Screenshot showing Hyper-V with nested SQL Server VM](./hyperv-nested-vm.png)
+
+  ![Screenshot showing Azure nested SQL Server VM afeter login](./hyperv-nested-vm-sql-login.png)
+
+- Open Microsoft SQL Server Management Studio (a Windows shortcut will be created for you) and validate the *AdventureWorksLT2019* sample database is deployed.
+
+  ![Screenshot showing SQL Management Studio](./sqlserver-management-login.png)
+
+  ![Screenshot showing SQL Management Studio](./sqlserver-management-database.png)
+
+## Microsoft Defender for Cloud - SQL servers on machines
+
+This section guides you through different settings for enabling Microsoft Defender for Cloud - SQL servers on machines. Most of these settings are already enabled during the logon script execution when login to _JS-Client_ Azure VM. Even though these are pre-configured there might be delays in showing them in the Azure portal.
+
+- Following are the settings of Microsoft Defender for Cloud - SQL servers on machines configured using automation scripts and can be reviewed in Azure portal.
+
+  ![Screenshot showing Microsoft Defender for Cloud plans](./microsoft-defender-plans.png)
+
+  ![Screenshot showing Microsoft Defender for Cloud SQL enabled](./defender-sql-plan.png)
+
+- Following screen shows Arc-enabled SQL server Defender for Cloud enablement and protection status. Defender for Cloud for SQL Server is enabled at the subscription level, but protection status is still showing is not enabled. Please note it may take sometime to show this status in Azure portal, but still able to detect SQL threats generated by the test scripts.
+
+  ![Screenshot showing Microsoft Defender for Cloud - Arc-enabled SQL server status](./arcsql-defender-status.png)
+
+- Following screen shows the SQL threat detected by Defender for Cloud.
+
+  ![Screenshot showing Defender for SQL security incidents and alerts](./defender-sql-security-incidents.png)
+
+- Following is the test script used to generate SQL threats, detect, and alert using Defender for Cloud for SQL servers.
+
+  ![Screenshot showing Defender for SQL test scripts](./defender-sql-testing-script.png)
+
+- Please note once in a while these test execution may fails randomly. If you don't find these alerts, login to nested SQL VM in Hyper-V and execute test script manually as show below.
```

Comment:
Why it is failing?

Reply:
There are some bugs in the PS commands. This is what I have escalated to the engineering team review and fix. This is not a showstopper for now. New script is consistently generating alerts and good with the scenario.